SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/SubJob/BrokerProfileOptOutSubJob.swift

```diff
@@ -40,7 +40,7 @@ struct BrokerProfileOptOutSubJob {
 
     public func runOptOut(for extractedProfile: ExtractedProfile,
                           brokerProfileQueryData: BrokerProfileQueryData,
-                          showWebView: Bool = false,
```

Comment:
I think in general we should be much more careful with default values (and avoid them much more often) to avoid bugs like this